src/test/java/edu/cornell/kfs/coa/document/validation/impl/GlobalIndirectCostRecoveryAccountsRuleTest.java

```diff
@@ -0,0 +1,103 @@
+package edu.cornell.kfs.coa.document.validation.impl;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import junit.framework.TestCase;
+
+import org.kuali.kfs.coa.businessobject.AccountGlobal;
+import org.kuali.kfs.coa.businessobject.AccountGlobalDetail;
+import org.kuali.kfs.coa.businessobject.IndirectCostRecoveryAccount;
+
+import edu.cornell.kfs.coa.businessobject.SubAccountGlobal;
+import edu.cornell.kfs.coa.businessobject.SubAccountGlobalDetail;
+import edu.cornell.kfs.coa.fixture.AccountGlobalDetailFixture;
+import edu.cornell.kfs.coa.fixture.AccountGlobalFixture;
+import edu.cornell.kfs.coa.fixture.SubAccountGlobalDetailFixture;
+import edu.cornell.kfs.coa.fixture.SubAccountGlobalFixture;
+
+public class GlobalIndirectCostRecoveryAccountsRuleTest extends TestCase {
+	
+    private static org.apache.log4j.Logger LOG = org.apache.log4j.Logger.getLogger(GlobalIndirectCostRecoveryAccountsRuleTest.class);
+	private GlobalIndirectCostRecoveryAccountsRule globalIndirectCostRecoveryAccountsRule;
+	
+	@Override
+	protected void setUp() throws Exception {
+		super.setUp();		
+
+		globalIndirectCostRecoveryAccountsRule = new GlobalIndirectCostRecoveryAccountsRule();
+	}
+	
+	public void testAccountGlobalCheckICRAccuntTotalDistributionOnDetailWillBe100PercentAfterUpdate_Pass() {
+		AccountGlobal accountGlobal = (AccountGlobal)AccountGlobalFixture.ACCT_GLOBAL_3333333_100_ACTIVE_1111111_2222222_98_2_INACTIVE.getAccountGlobal();
+		AccountGlobalDetail accountGlobalDetail = AccountGlobalDetailFixture.ACCOUNT_GLOBAL_DETAIL_1111111_2222222_98_2.getAccountGlobalDetail();
+		boolean result = globalIndirectCostRecoveryAccountsRule.checkICRAccuntTotalDistributionOnDetailWillBe100PercentAfterUpdate(
+				accountGlobal.getIndirectCostRecoveryAccounts(),
+				accountGlobalDetail.getAccount().getIndirectCostRecoveryAccounts(), 
+				accountGlobalDetail, accountGlobal);
+		LOG.info("Check Account Global updated ICR accounts distribution is 100%: " + result);
+		assertTrue("Updated Account Global ICR account distribution is 100%", result);
```

Comment:
I would recommend updating the assertion messages in this class to replace "is" with "should have been" or equivalent, to avoid confusion if a failure occurs. Also, I don't think you need the LOG.info() calls since the assertions should already validate and clarify the values.